en/docs/concepts/oidc.md

```diff
@@ -0,0 +1,53 @@
+# Introduction to OIDC
+
+### What is OIDC?
+OIDC is an authentication protocol which builds on top of the OAuth2.0 protocol. This is not a replacement to OAuth2.0, 
+but an extension where the clients can request by including the 'openid' scope value in the Authorization Request. 
+This protocol verifies user identity by authenticating the end user against the Authorization Server.
+
+### Why OIDC?
+When authenticating to an online system, a user should be able to prove his identity to the system. 
+A single user can have multiple identites (Passport number, NIC number, finger print) and proving identity to a system
+also can be done in many ways such as using credentials, biometric information or any other mechanism. OpenID Connect provides a 
+trustworthy, simple and effective mechanism for individuals to identify themselves to many application and service 
+providers using one or more of their identities stored at a trusted identity provider (IdP).
+
+OIDC provides a single login to multiple sites through an Identity Provider. It provides secure access to a client or an
+application without the user sharing the credentials of the application by exchanging tokens. Apart from that the clients
+receive user identity related details in the format of jwt which is very easy to process and maintain. Even though this 
+protocol is simple it has more features that match the enterprise demand supporting a wider variety of client types as 
```

Comment:
Even though this protocol is simple {insert comma}, it has more features

Reply:
supporting a wider variety of client types **such** as web, mobile, desktop {insert comma}, and **JavaScript** clients